doc/backing-store-crd.md

```diff
@@ -80,3 +92,209 @@ status:
+kind: BackingStore
+metadata:
+  finalizers:
+  - noobaa.io/finalizer
+  labels:
+    app: noobaa
+  name: jenia
+  namespace: jenia
+spec:
+  googleCloudStorage:
+    secret:
+      name: backing-store-google-cloud-storage-jenia
+      namespace: jenia
+    targetBucket: jenia
+  type: google-cloud-storage
+status:
+  conditions:
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "True"
+    type: Available
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "False"
+    type: Progressing
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "False"
+    type: Degraded
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "True"
+    type: Upgradeable
+  phase: Ready
+```
+
+S3-COMPATIBLE-TYPE:
+```shell
+nb -n jenia backingstore create s3-compatible jenia --endpoint http://jenia.com --signature-version v4 --access-key KEY --secret-key SECRET --target-bucket jenia
+```
+```yaml
+apiVersion: noobaa.io/v1alpha1
+kind: BackingStore
+metadata:
+  finalizers:
+  - noobaa.io/finalizer
+  labels:
+    app: noobaa
+  name: jenia
+  namespace: jenia
+spec:
+  s3Compatible:
+    endpoint: http://jenia.com
+    secret:
+      name: backing-store-s3-compatible-jenia
+      namespace: jenia
+    signatureVersion: v4
+    targetBucket: jenia
+  type: s3-compatible
+status:
+  conditions:
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "True"
+    type: Available
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "False"
+    type: Progressing
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "False"
+    type: Degraded
+  - lastHeartbeatTime: "1337-04-26T00:00:00Z"
+    lastTransitionTime: "1337-04-26T00:00:00Z"
+    message: noobaa operator completed reconcile - backing store is ready
+    reason: BackingStorePhaseReady
+    status: "True"
+    type: Upgradeable
+  phase: Ready
+```
```

Comment:
Suggest that the status section be removed from the above yamls since it is not something a user would specify as part of a CR.